rust/src/lib/ifaces/bond.rs

```diff
@@ -0,0 +1,475 @@
+use crate::{BaseInterface, ErrorKind, InterfaceType, NmstateError};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+#[serde(rename_all = "kebab-case")]
+pub struct BondInterface {
+    #[serde(flatten)]
+    pub base: BaseInterface,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub link_aggregation: Option<BondConfig>,
```

Comment:
This name is really annoying when typing. Could you change it to `bond` and use:

`#[serde(skip_serializing_if = "Option::is_none", rename = "link_aggregation")]`

Reply:
Sure, done! Thanks.